cmd/register/register_util.go

```diff
@@ -222,6 +240,33 @@ func hydrateTaskSpec(task *admin.TaskSpec, sourceCode string) error {
 				task.Template.GetContainer().Args[k] = string(remotePath)
 			}
 		}
+	} else if task.Template.GetK8SPod() != nil && task.Template.GetK8SPod().PodSpec != nil {
+		var podSpec = v1.PodSpec{}
+		err := utils.UnmarshalStructToObj(task.Template.GetK8SPod().PodSpec, &podSpec)
+		if err != nil {
+			return err
+		}
+		for containerIdx, container := range podSpec.Containers {
```

Comment:
nit : nil check for podSpec.Containers and container.Args unless thats always populated

Reply:
shouldn't be necessary, this just won't range over them if they're nil